File: official/docs/java/users/UpdateChildUser.java
package users;

import java.util.HashMap;

import com.easypost.EasyPost;
import com.easypost.exception.EasyPostException;
import com.easypost.model.User;

public class UpdateChildUser {
    public static void main(String[] args) throws EasyPostException {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");

        HashMap<String, Object> params = new HashMap<String, Object>();
        params.put("name", "Test Child");

        User user = User.retrieveMe();

        user.update(params);

        System.out.println(user);
    }
}

File: official/docs/go/users/update_child_user.go
package example

import (
	"fmt"
	"os"

	"github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
	client := easypost.New(apiKey)

	user, _ := client.GetUser("user_...")
	name := "Test Child"

	user, _ = client.UpdateUser(
		&easypost.UserOptions{
			Name: &name,
		},
	)

	fmt.Println(user)
}
